tests/conftest.py

```diff
@@ -0,0 +1,111 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+import os
+import subprocess
+
+import pytest
+
+from airflow.utils import db
+
+
+@pytest.fixture()
+def reset_environment():
+    """
+    Resets env variables.
+    """
+    init_env = os.environ.copy()
+    yield
+    changed_env = os.environ
+    for key in changed_env:
+        if key not in init_env:
+            del os.environ[key]
+        else:
+            os.environ[key] = init_env[key]
+
+
+@pytest.fixture()
+def reset_db():
+    """
+    Resets Airflow db.
+    """
+    db.resetdb()
+    yield
+
+
+def pytest_addoption(parser):
+    """
+    Add options parser for custom plugins
+    """
+    group = parser.getgroup("airflow")
+    group.addoption(
+        "--with-db-init",
+        action="store_true",
+        dest="db_init",
+        help="Forces database initialization before tests",
+    )
+
+
+@pytest.fixture(autouse=True, scope="session")
```

Comment:
This is FANTASTIC!

Reply:
Moreover if everything passes then there's no log from db.reset ✅

Reply:
This is great! 👍